lib/theme_check/checks/translation_key_exists.rb

```diff
@@ -5,18 +5,31 @@ class TranslationKeyExists < LiquidCheck
     category :translation
     doc docs_url(__FILE__)
 
+    def initialize
+      @schema_locales = {}
+      @nodes = []
```

Comment:
Just remembered that Check instances are reused between files. Might want to do something like UnusedAssign and have nodes per file.

```
def initialize
  @nodes = {}
end

def on_document(node)
  @nodes[node.theme_file.name] = []
end

def on_variable(node)
  #...
  @nodes[node.theme_file.name] << node
end

# etc.
```

Reply:
Otherwise the @nodes array will keep growing as more files are checked.

Reply:
Yea, this makes sense.